src/coreclr/vm/contractimpl.cpp

```diff
@@ -434,9 +434,11 @@ DispatchMap::CreateEncodedMapping(
             curTargetSlot = -1;
             do
             {
+                /* TODO: right now it's unclear to me whether the SVMs should go in the method table or not, for now I assume they should.
```

Comment:
My initial thought was to store this sort of detail in a parallel table to the existing scheme, but we'll want to take a close look here to see if this is the right approach.